doc/howtos/pull-request-guidelines.rst

```diff
@@ -0,0 +1,161 @@
+.. _pull-request-guidelines:
+
+Ostro |trade| OS Pull Request Guidelines
+########################################
+
+The :ref:`contributor-guide` explains how to submit changes as pull
+requests (PRs) and how they will get merged.
+
+This document complements those instructions with more hands-on
+guidelines about what information will be needed during a review, what
+kind of mistakes to watch out for during review, and how to deal with
+PRs.
+
+The goal is to make the review process more consistent and help less
+experienced reviewers in catching potential mistakes that overwise
+might get overlooked. A submitter can use this to avoid these mistakes
+before even submitting the pull request.
+
+Workflow
+========
+
+PRs are typically checked in this order:
+
+1. pull request message
+2. commit messages
+3. actual diff
+
+Reviewers, beware that jumping straight to a diff may miss relevant
```

Comment:
Reviewers should look at the entire file and not just review the diffs to avoid missing relevant explanations..